modules/gapi/src/backends/fluid/gfluidbuffer.cpp

```diff
@@ -181,7 +181,7 @@ const uint8_t* fluid::BorderHandlerT<cv::BORDER_CONSTANT>::inLineB(int /*log_idx
 void fluid::BorderHandlerT<cv::BORDER_CONSTANT>::fillCompileTimeBorder(BufferStorageWithBorder& data)
 {
     m_const_border.create(1, data.cols(), data.data().type());
-    m_const_border = m_border_value;
+    m_const_border = {m_border_value[0], m_border_value[1], m_border_value[2], m_border_value[3]}; // crutch is waiting deowned mat
```

Comment:
Without "deowned" `Mat` we get exception here. We use `cv::gapi::own::Mat`'s `operator=` for `cv::Scalar`, but we haven't `operator=` for this case. It will be resolved when `Mat` changes.

"to_own crutch" now. [39 string](https://github.com/opencv/opencv/blob/9bc5ae37ac14542cf8cd95bb3366d8454d61db17/modules/gapi/include/opencv2/gapi/own/convert.hpp)

Reply:
Please align this code properly